### PR TITLE
Bump CMake version to avoid CMP0048 warning

### PR DESCRIPTION
**Pull request by ahcorde**
This bumps the minimum CMake version to 3.0.2, which is the [minimum supported by ROS Kinetic](https://www.ros.org/reps/rep-0003.html#kinetic-kame-may-2016-may-2021) and new enough to default to the NEW behavior of [CMP0048](https://cmake.org/cmake/help/v3.0/policy/CMP0048.html). This avoids a CMake warning when building and testing this package in Debian Buster and Ubuntu Focal.

ros/catkin#1052

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

**Review by sloretz — APPROVED** [review]

**Comment by sloretz** [comment]
@nuclearsandwich mind taking a look at this one? I think you might be the maintainer of this repo.

**Comment by nuclearsandwich** [comment]
> @nuclearsandwich mind taking a look at this one? I think you might be the maintainer of this repo.

I may have forgotten but I can't deny the possibility. I'll take a look.

**Review by nuclearsandwich — APPROVED** [review]

**Comment by nuclearsandwich** [comment]
@ahcorde you can merge this or I'll do it when I make the next metapackages release. I'm working on a rosdep release today.

**Comment by ahcorde** [comment]
feel free to merge it when you are ready to release the next metapackages release.

**Comment by sloretz** [comment]
@nuclearsandwich @ahcorde I'm clicking merge to remove this PR from my list; hope you don't mind :)

**Comment by nuclearsandwich** [comment]
> I'm clicking merge to remove this PR from my list; hope you don't mind :)

Not at all. Thanks.

**Pull request merged**
